### PR TITLE
Strutil: HammingDistance func

### PR DESCRIPTION
**Pull request by donutloop**
The Hamming distance is the number of positions at which the corresponding symbols are different

### CODE REVIEW

**Comment by donutloop** [comment]
@duke-git Please review. Do you believe we should merge this no breaking change into rc or main?

File: docs/api/packages/strutil.md [hunk 1]
```diff
@@ -1495,4 +1496,36 @@ func main() {
     // abc
     // bc
 }
+```
+
+### <span id="HammingDistance">HammingDistance</span>
+
+<p>TBD</p>
```
Comment (donutloop) [inline docs/api/packages/strutil.md:1503]: @duke-git Please change this

**Comment by duke-git** [comment]
> @duke-git Please review. Do you believe we should merge this no breaking change into rc or main?

@donutloop, yes, please merge this PR to `rc` branch. `main` branch is frozen, won't accept PR again.

**Comment by donutloop** [comment]
> > @duke-git Please review. Do you believe we should merge this no breaking change into rc or main?
> 
> @donutloop, yes, please merge this PR to `rc` branch. `main` branch is frozen, won't accept PR again.

@duke-git 
Updated branch

**Review by duke-git — APPROVED** [review]

**Pull request merged**